src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion_img2img.py

```diff
@@ -64,7 +64,6 @@
 """
 
 
-# Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_img2img.preprocess
```

Comment:
Let's leave the code in the original img2img for now and deprecate it (some people might also import this function). This way we can also keep the copied from